lib/comment.js

```diff
@@ -48,6 +59,15 @@ module.exports = function(number, metadata) {
         // if there are collaborators, we ask them for a review
         if (metadata.reviewersExcludingAuthor.length > 0 || metadata.isRoot) {
             if (metadata.missingReviewers.length > 0) {
+                if (dryRun) {
+                    console.log(`[DRY-RUN] Would add reviewers ${metadata.missingReviewers} to issue ${number}`);
+                    // Assume that adding reviewers would have succeeded.
+                    if (!metadata.isMergeable) {
+                        console.log(`[DRY-RUN] Would add label "status:needs-collaborator" and comment "No reviewer on this pull request has write-access to the repository." to issue ${number}`);
```

Comment:
Put the label and message in variables, and change "issue" to "PR".

Similarly below.